app/src/main/java/net/skyscanner/backpack/demo/compose/BackpackPreview.kt

```diff
@@ -37,11 +38,9 @@ fun BackpackPreview(
 ) {
   BackpackDemoTheme {
     CompositionLocalProvider(*providers) {
-      Surface(
-        color = background.takeOrElse { BpkTheme.colors.background },
-        modifier = modifier,
-        content = content,
-      )
+      Box(modifier.background(background.takeOrElse { BpkTheme.colors.background })) {
```

Comment:
using surface was causing a few more issues (cutting things off, unintentionally centering the screenshot tests), so switching to box instead